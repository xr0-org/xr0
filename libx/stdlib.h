#ifndef STDLIB_H
#define STDLIB_H

axiom void *
malloc(int size) [ .alloc result; ];

axiom void
free(void *ptr) [ .dealloc ptr; ];

#endif
