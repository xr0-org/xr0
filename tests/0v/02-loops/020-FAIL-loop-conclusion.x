#include <stdlib.h>

void
unit()
{
	int i;

	for (i = 0; i < 9; i++) ~ [ i = [0?10]; ]
		;

	~ [ i == 8; ]
}
