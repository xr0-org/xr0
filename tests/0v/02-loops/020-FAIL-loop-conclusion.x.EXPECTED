02-loops/020-FAIL-loop-conclusion.x:11:14: cannot verify statement (unit | #0 âˆˆ {9})
